Orange/widgets/data/tests/test_owpythonscript.py

```diff
@@ -68,3 +68,23 @@ def test_wrong_outputs(self):
             self.assertTrue(getattr(self.widget.Error, _output).is_shown())
             self.send_signal(_input, data)
             self.assertFalse(getattr(self.widget.Error, _output).is_shown())
+
+    def test_multiple_data_inputs(self):
```

Comment:
Could the test be expanded to test more of widget's functionality, like what happens with these multiple objects?